packages/@lwc/integration-karma/helpers/test-utils.js

```diff
@@ -577,19 +576,29 @@ window.TestUtils = (function (lwc, jasmine, beforeAll) {
                     expect(calls).toHaveSize(matchers.length);
                     for (let i = 0; i < matchers.length; i++) {
                         const matcher = matchers[i];
-                        const call = calls[i][0];
-                        const message = typeof call === 'string' ? call : call.message;
-                        if (typeof matcher === 'string') {
-                            expect(message).toContain(matcher);
-                        } else {
-                            expect(message).toMatch(matcher);
-                        }
+                        const args = calls[i];
+                        const argsString = args.map((arg) => stringifyArg(arg)).join(' ');
+                        expect(argsString).toMatch(matcher);
                     }
                 }
             }
         };
     }
 
+    function stringifyArg(arg) {
```

Comment:
Better way to do this? I couldn't decide between just printing the `nodeName` Or the whole Node. Printing the node doesn't seem to add much to the tests and just makes the output messier?